### PR TITLE
SP-1 Added route and migration

### PR DESCRIPTION
**Pull request by E-zh**
Added migration and route.

### CODE REVIEW

File: src/Http/routes.php [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?php
+
+/**
+ * Routes here...
+ */
```
Comment (Anonim-khb) [inline src/Http/routes.php]: You need add empty string

File: src/StaticPagesServiceProvider.php [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+use Illuminate\Support\ServiceProvider;
+
+class StaticPagesServiceProvider extends ServiceProvider
+{
+
+    public function boot(){
+        require __DIR__ . '/Http/routes.php';
+    }
+
+}
```
Comment (Anonim-khb) [inline src/StaticPagesServiceProvider.php]: You need add empty string

File: src/database/migrations/2016_04_20_create_staticpages_table.php [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::drop('static-pages');
+    }
+}
```
Comment (Anonim-khb) [inline src/database/migrations/2016_04_20_create_staticpages_table.php]: You need add empty string

File: .gitignore [hunk 4]
```diff
@@ -0,0 +1,3 @@
+/vendor/
+/.idea/
+/composer.phar
```
Comment (Anonim-khb) [inline .gitignore]: Add empty string.

File: src/StaticPagesServiceProvider.php (hunk 2 above)
Comment (garf) [inline src/StaticPagesServiceProvider.php]: @Anonim-khb do you mean "EOL"? :)

File: .gitignore (hunk 4 above)
Comment (garf) [inline .gitignore]: we don't need this in gitignore. Just remove it totally from git using `git rm composer.phar`

**Pull request merged**